### ISSUE TITLE
Grafana board - Gateway panel empty

### ISSUE THREAD

**Issue by RobertRad** [camunda/zeebe#9443]
**Describe the bug**

The Grafana board has no data for the Gateway.

**To Reproduce**

- Open a Grafana instance
- Import a dashboard with the JSON from [zeebe.json](https://github.com/camunda/zeebe/blob/main/monitor/grafana/zeebe.json)
- Make sure data is coming in
- Open the "Gateway" row
- All 3 panels show "No data points" or "No data"

**Expected behavior**
There should be a meaningful diagram

**Details**

For the first panel (Gateway Request Latency) the query is `sum(increase(zeebe_gateway_request_latency_bucket{namespace=~"$namespace",partition=~"$partition"}[30s])) by (le)`

This leaves an empty in the metrics browser.

When removing the `increase(...[30s])`, there is data.
`sum(zeebe_gateway_request_latency_bucket{namespace=~"$namespace",partition=~"$partition"}) by (le)`

**Environment:**
The helm chart "camunda-platform" from "https://helm.camunda.io" in version "8.0.5".
Grafana version (Grafana cloud): v8.5.2

**Comment by Zelldon** [comment]
Hey @RobertRad 

could you provide the values file your use, if any? Do you see other metrics? Do you use a different zeebe version then the default?

I'm not able to reproduce your issue, we have in our weekly benchmarks metrics and use the most recent helm chart versions. 

Greets
Chris

**Comment by RobertRad** [comment]
We have updated now to Chart version 8.0.8 - the data looks the same.
The zeebe version used by this chart seems to be 8.0.0 - The controller_revison_hash is `camunda-zeebe-6554bb5b9b`.

Here a screenshot of our Overview chart:
![camunda-zeebee-grafana-chart-overview](https://user-images.githubusercontent.com/11713844/170701107-ca450d00-4a83-4725-8b9e-8b250db4189e.png)
There is also a number of data missing.

An excerpt of our values file:
```yaml
# Zeebe configuration for the Zeebe sub chart. Contains configuration for the Zeebe broker and related resources.
zeebe:
  nodeSelector: { }
  tolerations: [ ]
  affinity: null

# Gateway configuration to define properties related to the standalone gateway
zeebe-gateway:
  nodeSelector: { }
  tolerations: [ ]
  affinity: null
```
So nothing special here.
The most part of our config are the ingress for operate & tasklist. Besides that is mostly the comments from the template.

**Comment by RobertRad** [comment]
Maybe it has to do with metrics exporter?
This is our Grafana Agent config:
```
    - job_name: camunda-zeebe
      metrics_path: /metrics
      kubernetes_sd_configs:
      - role: pod
      relabel_configs:
        - action: labelmap
          regex: __meta_kubernetes_pod_label_(.+)
        - action: keep
          source_labels: [app]
          regex: ^camunda-platform$
        - action: drop
          source_labels : [__meta_kubernetes_pod_container_port_number]
          regex: '(4143|4191|80)'
        - action: replace
          source_labels: [__meta_kubernetes_namespace]
          target_label: namespace
        - action: replace
          source_labels: [__meta_kubernetes_pod_name]
          target_label: pod
        - action: replace
          source_labels: [__meta_kubernetes_pod_label_linkerd_io_proxy_deployment]
          target_label: deployment
```
The `keep`-Action should work for zeebe, zeebe-gateway & even operate.
The `drop`-Action drops our linkerd-Containers.
Everything else should be straight forward.

Any ideas, how I can approach the problem of the incomplete Grafana data?

**Comment by Zelldon** [comment]
Hey @RobertRad sorry for the late response, but just to clarify. Did you set `prometheusServiceMonitor.enabled=true` ? Because otherwise no servicemonitor is deployed with the helm charts.

**Comment by RobertRad** [comment]
@Zelldon The `prometheusServiceMonitor` wasn't the problem, we scraped metrics the with our own Prometheus.

But the file lead to the solution of our problem. We were scraping with a resolution of 60sec. In this case the `rate(some_metric{namespace=~"$namespace"}[1m])` doesn't work. So the resolution must be smaller than that.
I opened a PR (https://github.com/camunda/camunda-platform-helm/pull/331) which documents this in the helm chart.

The dashboard does not show all data, yet. But this fix added a lot of metrics!
